Sources/StreamChatUI/ChatMessageList/RETHINK/MessageThreadVC.swift

```diff
@@ -0,0 +1,323 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+import UIKit
+
+class MessageThreadVC<ExtraData: ExtraDataTypes>: _ViewController, UICollectionViewDelegate, UICollectionViewDataSource,
+    UIConfigProvider {
+    var channelController: _ChatChannelController<ExtraData>!
```

Comment:
I don't think we ween the channel controller

Reply:
I think it's needed in `MessageComposerVC`